### ISSUE TITLE
No Error/Warning when certificates are not trusted 

### ISSUE THREAD

**Issue by mtamboli** [eclipse-che/che#16713]
### Describe the bug

Code Read Workspaces/Che are enabled with TLS enabled setting in Kabanero 0.6.0. If user does not extract and trust this certificate, user can login fine but CRW keeps on spinning and does not show workspace create screen. I would expect it to let me know the problem so that user can fix the certificates.
I would have expected at least SSL Handshake error or something like that.

Also, is trusting the self signed certificate on the browser not adequate?

### Che version

 - [ ] latest
 - [ ] nightly
 - [x] other: please specify
```
2.1.0.GA :: crw_master/1107 :: che-dev @ 9348 (19) :: che-parent @ 29042 (7.9.2) :: che @ 0ea1a6 (7.9.2) :: codeready-workspaces @ 5661 (2.1.0.GA)
```
### Steps to reproduce
1. Enable CRW in Kabanero 0.6.0/OCP 4.3 Cluster with SSL enabled.
```
codeReadyWorkspaces:
      enable: true
      operator:
        customResourceInstance:
          devFileRegistryImage: {}
          selfSignedCert: true
          tlsSupport: true
```
2. You can access the CRW route but it is not able to load any further and there are no errors indicating the cause.

![image](https://media.github.ibm.com/user/8554/files/f64b8f80-7a79-11ea-9c3c-97854cd1c9a9)

3. Problem is resolved if you trust the root certificate of the cluster. 

### Expected behavior

I expected opportunity to trust self signed certificate from the browser. At the least, it should display the error when certificate is not trusted instead of just spinning for long time. This does not give user a good experience as the user does not know what is wrong.
 
### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [x] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl 
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know
Kabanero - https://github.com/kabanero-io/kabanero-foundation
### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [x] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by tolusha** [comment]
@mtamboli 
Correct me if I am wrong.
1. Eclipse Che is deployed with tls enabled.
2. User log in into Eclipse Che (without importing certificate into a browser)
3. User can't start a workspace
right?

**Comment by mtamboli** [comment]
> @mtamboli
> Correct me if I am wrong.
> 
> 1. Eclipse Che is deployed with tls enabled.
> 2. User log in into Eclipse Che (without importing certificate into a browser)
> 3. User can't start a workspace
>    right?

Yes, That is correct. It lets you login to CRW and then keeps spinning. You have to trust the cluster certificate before you get to "create workspace" page. There is no way to trust from the browser itself.

**Comment by tolusha** [comment]
@mtamboli 
I see. 
This kind of check and a proper notification of missing imported certificate have been added in recent version of Eclipse Che.
/cc
@sleshchenko

**Comment by sleshchenko** [comment]
The proper message is merged in https://github.com/eclipse/che/pull/16380 which is part of 7.11.0 not sure which CRW should include it.
cc @nickboldt

**Comment by mtamboli** [comment]
I can close this issue after verifying it in CRW. @johnmcollier do you know which CRW version will have version 7.11.0?

**Comment by jagraj** [comment]
I tried this on CodeReady 2.1.1 which is based on Che 7.9.3 release. We see this message when we try to access CRW home page. We did not set the certificate yet and the message looks good but the link taking us to the general doc home page but I am not sure where to look to configure certificates. We documented in Kabanero docs how to export and import certificates (https://kabanero.io/docs/ref/general/installation/installing-codeready-and-codewind.html#using-codeready-workspaces-with-self-signed-certificates). It would be nice the message in the screesnhot below take user to the certifcates configuration in CodeReady workspaces docs.

We are not sure about the fix coming in 7.11.0.?

![Screen Shot 2020-05-01 at 1 54 10 PM](https://user-images.githubusercontent.com/713849/80828860-0277c800-8bb4-11ea-832a-36cffe7e4e0b.png)
